### DISCUSSION TITLE
blocked account

### DISCUSSION THREAD

**Discussion by SAMISHAIKH801** [vercel/community#3746] [Account Issue] [answered]
### Question

Subject: Account Creation Issue on Vercel

Hello Vercel Support,

I hope this message finds you well. I am currently facing an issue while trying to create an account on Vercel. Whenever I attempt to sign up, I encounter an error and my account creation process is not successful.

The error message I'm receiving states: "We already have a registered user with this email address. Log in to connect your GitHub account." I have not been able to proceed further due to this issue.

Could you please assist me in resolving this problem? I've tried different email addresses and methods, but the issue persists. I would greatly appreciate your help in creating my Vercel account so that I can start using your platform for my projects.

Thank you for your attention and assistance.

Best regards,
samishaikh

**Comment by swarnava** [comment, marked as answer]
Hi,

Please email at registration@vercel.com
